tests/integration/Settings/__snapshots__/General_Tab_Test__should_match_general_tab_snapshots__all settings enabled__0.snapshot.html

```diff
@@ -1 +1,9 @@
+			This activates access to TEC AI chatbot and in-app priority support for premium users.
+			<br /><a href="https://evnt.is/1bcl">What permissions are being granted?</a>
+			<br /><a href="https://evnt.is/1bcm">Read our terms of service</a>.
+			<br /><a href="https://evnt.is/1bcn">Read our privacy policy</a>.</label><p class="tooltip description">Enable this option to share usage data with Event Tickets and StellarWP.
+			This activates access to TEC AI chatbot and in-app priority support for premium users.
+			<br /><a href="https://evnt.is/1bcl">What permissions are being granted?</a>
+			<br /><a href="https://evnt.is/1bcm">Read our terms of service</a>.
+			<br /><a href="https://evnt.is/1bcn">Read our privacy policy</a>.</p></div></fieldset></div>
```

Comment:
```suggestion
			This activates access to TEC AI chatbot and in-app priority support for premium users.
			<br /><a href="https://evnt.is/1bcl">What permissions are being granted?</a>
			<br /><a href="https://evnt.is/1bcm">Read our terms of service</a>.
			<br /><a href="https://evnt.is/1bcn">Read our privacy policy</a>.</label><p class="tooltip description">Enable this option to share usage data with Event Tickets and StellarWP.
			This activates access to TEC AI chatbot and in-app priority support for premium users.
			<br /><a href="https://evnt.is/1bcl">What permissions are being granted?</a>
			<br /><a href="https://evnt.is/1bcm">Read our terms of service</a>.
			<br /><a href="https://evnt.is/1bcn">Read our privacy policy</a>.</p></div></fieldset></div>
```

Should this links be opened in new window?